ENGAGEHF/Onboarding/InvitationCodeError.swift

```diff
@@ -0,0 +1,28 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+
+
+// Based on: https://github.com/StanfordBDHG/PediatricAppleWatchStudy/pull/54/files
```

Comment:
I would suggest to move these comments in the header of the file, maybe as an additional line between 7 and 8 👍

```swift
// SPDX-License-Identifier: MIT
//
// Based on: https://github.com/StanfordBDHG/PediatricAppleWatchStudy/pull/54/files
// 
```